sdk/python/v1beta1/kubeflow/katib/api/katib_report_metrics.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 The Kubeflow Authors.
```

Comment:
One idea might be to add this function to Katib utils: https://github.com/kubeflow/katib/blob/1d9be470dbf9499c6a94070572956ff844b1c66e/sdk/python/v1beta1/kubeflow/katib/utils/utils.py

And then add this function to [the `__init__.py file`](https://github.com/kubeflow/katib/blob/1d9be470dbf9499c6a94070572956ff844b1c66e/sdk/python/v1beta1/kubeflow/katib/__init__.py#L72-L75):

```py
from kubeflow.katib.utils.utils import report_metrics
```

So users can just use it as follows:
```python
import kubeflow.katib as katib

katib.report_metrics()
```


That will help us to reduce number of files.
WDYT @Electronic-Waste @johnugeorge @droctothorpe ?

Reply:
I think we should put `report_metrics` in the `api` dir because it's a user interface rather than a util function.

Reply:
Ok, than should we just call this file `report_metrics.py` and also add it to the `__init__.py` file as I mentioned above?

Reply:
Yeah, I agree with that.